v2/azure-arm.yaml

```diff
@@ -121,6 +121,16 @@ exportFilters:
     version: v*api20210401
     name: 'StorageAccountsBlobServicesContainer'
     because: "including StorageAccountsBlobServicesContainer resource"
+  - action: include-transitive
+    group: storage
+    version: v*api20210401
+    name: 'StorageAccountsQueueService'
+    because: "including StorageAccountsQueueService resource"
+  - action: include-transitive
+    group: storage
+    version: v*api20210401
+    name: 'StorageAccountsQueueServices*'
```

Comment:
If this is just adding the queue resource, why not specify it by name here? It made me suspicious. 🤨

Reply:
it's both `StorageAccountsQueueServices` and `StorageAccountsQueueServicesQueue`